shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -888,16 +895,31 @@ - (instancetype)initWithTarget:(id)target
 }
 
 - (void)touchesBegan:(NSSet*)touches withEvent:(UIEvent*)event {
-  [_platformViewsController->getFlutterViewController() touchesBegan:touches withEvent:event];
+  if (!_flutterViewController) {
+    // We try to get a reference to `_flutterViewController` before handling gestures.
+    // Once `_flutterViewController` is available, we keep this weak reference at least
+    // until the end of the touch event.
+    _flutterViewController = _platformViewsController->getFlutterViewController();
+    if (!_flutterViewController) {
+      return;
+    }
+  }
+  [_flutterViewController touchesBegan:touches withEvent:event];
   _currentTouchPointersCount += touches.count;
```

Comment:
What's the implication of not incrementing this counter if we have no VC?

Should we be setting to zero in the early return above?